packages/react-core/src/components/Modal/ModalBox.tsx

```diff
@@ -9,6 +9,10 @@ export interface ModalBoxProps extends React.HTMLProps<HTMLDivElement> {
   className?: string;
   /** Variant of the modal */
   variant?: 'small' | 'medium' | 'large' | 'default';
+  /** Alternate position of the modal */
+  position?: 'top';
+  /** Custom spacer distance from position */
```

Comment:
Add `Can be any valid CSS distance.`

Reply:
[Length](https://developer.mozilla.org/en-US/docs/Web/CSS/length) is generally how those values are referred to. Maybe "CSS length"?  Technically it would be "length/percentage" since [length-percentage](https://developer.mozilla.org/en-US/docs/Web/CSS/length-percentage) are a separate thing.